src/annotator/guest.ts

```diff
@@ -333,6 +333,16 @@ export class Guest extends TinyEmitter implements Annotator, Destroyable {
     this._hoveredAnnotations = new Set();
   }
 
+  /** Return true if the sidebar is shown alongside the page content. */
+  private _sideBySideActive(): boolean {
+    if (this.sideBySide?.mode === 'manual' && this.sideBySide.isActive) {
```

Comment:
The `mode` check is needed to narrow the union so that TypeScript knows `isActive` is a valid property. I do think it is helpful for the reader as well, even if not strictly essential.